src/RefactorCodeChallengeTests/RefactorCodeChallengeTests.cs

```diff
@@ -39,20 +42,19 @@ public void CalculateTotalLengthOfShips_GivenACollectionOfShips_ReturnsTheExpect
     [Test]
     public void GetNameOfLongestBattleship_GivenACollectionOfShips_ReturnsTheNameOfTheLongestBattleship()
     {
-        var expectedLongestBattleshipName = ShipOneName;
-
-        var sut = new ShipManager(_testCollectionOfShipsInput);
+        var sut = new ShipManager(TestCollectionOfShipsInput);
 
-        var actuallongestBattleshipName = sut.GetNameOfLongestBattleship();
+        var nameOfLongestBattleship = sut.GetNameOfLongestBattleship();
 
-        Assert.That(actuallongestBattleshipName, Is.EqualTo(expectedLongestBattleshipName));
+        Assert.That(nameOfLongestBattleship, Is.EqualTo(ShipOneName));
```

Comment:
I acutally like the actual and expected syntax, but that is probably my personal preference.

Reply:
I added these back.